StackExchange.Redis/StackExchange/Redis/RedisCommand.cs

```diff
@@ -38,6 +38,13 @@ enum RedisCommand
         FLUSHALL,
         FLUSHDB,
 
+        GEOADD,
```

Comment:
I haven't checked yet, but note that some of these look fine for replica, some look master-only; there's a method somewhere that defines which is which (needs to be done for command routing purposes)